### PR TITLE
Increases Schedule:Compact IDD fields

### PR DESCRIPTION
**Pull request by shorowit**
Increases Schedule:Compact fields in IDD from 4500 to 10000. Allows the IDFEditor to work on larger schedules.

### CODE REVIEW

File: idd/Energy+.idd.in [hunk 1]
```diff
@@ -4030,442 +4030,506 @@ Schedule:Compact,
...
- A4368,A4369,A4370,A4371,A4372,A4373,A4374,A4375,A4376,A4377,A4378,A4379,A4380,A4381,A4382, \note fields as indicated
- A4383,A4384,A4385,A4386,A4387,A4388,A4389,A4390,A4391,A4392,A4393,A4394,A4395,A4396,A4397, \note fields as indicated
- A4398,A4399,A4400,A4401,A4402,A4403,A4404,A4405,A4406,A4407,A4408,A4409,A4410,A4411,A4412, \note fields as indicated
- A4413,A4414,A4415,A4416,A4417,A4418,A4419,A4420,A4421,A4422,A4423,A4424,A4425,A4426,A4427, \note fields as indicated
- A4428,A4429,A4430,A4431,A4432,A4433,A4434,A4435,A4436,A4437,A4438,A4439,A4440,A4441,A4442, \note fields as indicated
- A4443,A4444,A4445,A4446,A4447,A4448,A4449,A4450,A4451,A4452,A4453,A4454,A4455,A4456,A4457, \note fields as indicated
- A4458,A4459,A4460,A4461,A4462,A4463,A4464,A4465,A4466,A4467,A4468,A4469,A4470,A4471,A4472, \note fields as indicated
- A4473,A4474,A4475,A4476,A4477,A4478,A4479,A4480,A4481,A4482,A4483,A4484,A4485,A4486,A4487, \note fields as indicated
- A4488,A4489,A4490,A4491,A4492,A4493,A4494,A4495,A4496,A4497,A4498,A4499,A4500; \note fields as indicated
+  A7,A8,A9,A10,A11,A12,A13,A14,A15,A16,A17,A18,A19,A20, \note fields as indicated
```
Comment (Myoldmopar) [inline idd/Energy+.idd.in]: Fields 7-152 are probably quite often used, and as such it seems like you wouldn't want to remove the field names from them.  Is there a reason you needed to do that?

Comment (shorowit) [reply idd/Energy+.idd.in]: No, that was inadvertent. I'll revert those field name changes, thanks for pointing it out.

**Comment by nrel-bot-2b** [comment]
@shorowit it has been 37 days since this pull request was last updated.

**Comment by nrel-bot-2b** [comment]
@shorowit it has been 7 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
I guess I don't really have any issue with this change.  @mjwitte ?

I'll pull up the latest develop into this branch and see if CI is happy with it.

**Comment by mjwitte** [comment]
@Myoldmopar This is ok.

**Comment by Myoldmopar** [comment]
So far so good.  If I see this coming up all green before I sleep I'll merge it in, otherwise it will drop early in the morning.

**Comment by Myoldmopar** [comment]
I'm not sure why CI status hasn't been refreshed for the custom_check case.  It completed earlier.  There was one warning, the PR didn't have an appropriate label.  I put NewFeature, but I don't really care what we label it, it just needs one of the valid ones.  Anyway, I'll still keep an eye on it tonight.

**Comment by Myoldmopar** [comment]
I am concerned that CI is having trouble posting back on this PR.  It's still happy everywhere else and there's nothing suspicious about the changes here.  I built the unit tests locally and there does seem to be one failing.  I'm rebuilding develop branch and will check whether it is failing there.

**Comment by Myoldmopar** [comment]
OK, CI is finally happy with this.  Not sure what was going on yesterday.  Anyway, merging this.

**Pull request merged**